### DISCUSSION TITLE
terraform

### DISCUSSION THREAD

**Discussion by sirishaelluri** [community/community#112694] [GitHub Education]
### Hi everyone,

hii welcome

**Comment by gil-son** [comment]
Hello,

What's the question?

**Comment by samus-aran** [comment]
Hi @sirishaelluri ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!h
